src/api/entities/Proposal/types.ts

```diff
@@ -51,10 +20,25 @@ export enum ProposalStage {
   Ended = 'Ended',
 }
 
-export interface ProposalDetails {
-  state: ProposalState;
-  module: string;
+export interface Call {
   method: string;
+  module: string;
+}
+
+export interface ProposalDetails {
+  pipId: number;
+  proposer: string;
+  createdAt: number;
```

Comment:
What is this? A block number? Timestamp?

Reply:
block number

Reply:
Should be BigNumber then